modules/ROOT/pages/fd-reuse-config.adoc

```diff
@@ -6,11 +6,13 @@ endif::[]
 
```

Comment:
INSTRUCTIONS FOR EDIT

can you review all the green on this PR? You already reviewed it, but a typo was discovered so while I had the file open, I made a couple of edits.  It's not a super-rush, but when you can would be great